src/api/eventTypes.js

```diff
@@ -17,7 +17,7 @@ import type { Message, Stream } from './apiTypes';
 //
 
 /** See ClientPresence, and the doc linked there. */
-export type UserStatus = 'active' | 'idle' | 'offline';
+export type PresenceStatus = 'active' | 'idle' | 'offline';
```

Comment:
This is a good rename! I think in fact it belongs at the very beginning of the branch -- then that properly clears things up for starting to use the term "user status" for this new feature.

Ditto the other related rename.